heartbeat/findif.sh

```diff
@@ -233,24 +233,19 @@ findif()
     fi
     case $1 in
     */*) : OK ;;
+    # "ip route" doesnt show netmask for IPv6 /128
+    *:*:*) : OK ;;
     *)
       ocf_log err "Unable to find cidr_netmask."
       return $OCF_ERR_GENERIC ;;
     esac
   fi
   [ -z "$nic" ] && nic=$3
   [ -z "$netmask" ] && netmask=${1#*/}
-  if [ $family = "inet" ] ; then
-    if [ -z "$brdcast" ] ; then
-      if [ -n "$7" ] ; then
-        set -- `ip -o -f $family addr show | grep $7`
-        [ "$5" = brd ] && brdcast=$6
-      fi
-    fi
-  else
-    if [ -z "$OCF_RESKEY_nic" -a "$netmask" != "${1#*/}" ] ; then
-      ocf_log err "Unable to find nic, or netmask mismatch."
```

Comment:
Is there any reason to totally eliminate this validation check?
It should detect a wrong configuration for the netmask parameter (which is the case 13 in IPaddr2v6 ocft test cases).

I think that this validation check is still useful and should be retained somehow so that cooperate with prefix /128,
but if you have a good reason to remove it, that might be ok. In that case, we would need to modify the IPaddr2v6 ocft script along with it with some explanations.

Reply:
Do you have any suggestions for how to make it work with /128?